static-assets/components/cstudio-forms/controls/date-time.js

```diff
@@ -1210,15 +1210,18 @@ YAHOO.extend(CStudioForms.Controls.DateTime, CStudioForms.CStudioFormField, {
 				currentDate.setDate(currentDate.getDate()+(modifier*value*daysInWeek))
 			}else if(type=="days") {
 				currentDate.setDate(currentDate.getDate()+(modifier*value));
+			}else if(type=="hours") {
+				currentDate.setTime(currentDate.getTime() + (modifier*(value*60*60*1000)));
```

Comment:
Change all these to triple equals

Multiply against the static resulting value (3,600,000) instead of performing the open every time